irohad/torii/processor/impl/transaction_processor_impl.cpp

```diff
@@ -158,6 +159,22 @@ namespace iroha {
       pcs_->propagate_transaction(transaction);
     }
 
+    void TransactionProcessorImpl::transactionSequenceHandle(
+        const shared_model::interface::TransactionSequence
+            &transaction_sequence) {
+      for (const auto &batch : transaction_sequence.batches()) {
+        if (batch.hasAllSignatures()) {
+          pcs_->propagate_batch(batch);
+        } else {
+          // TODO kamilsa 16.07.18 propagate full batch to mst when its
+          // interface is updated
+          for (const auto tx : batch.transactions()) {
```

Comment:
No reference as well, is there a reason?

Reply:
Yes! It is shared_ptr, which do not need to be passed by reference